src/AutosuggestContainer.js

```diff
@@ -69,13 +69,14 @@ export default class AutosuggestContainer extends Component {
     renderSectionTitle: PropTypes.func,
     getSectionSuggestions: PropTypes.func,
     focusInputOnSuggestionClick: PropTypes.bool,
+    selectFirstSuggestion: PropTypes.bool,
     theme: PropTypes.object,
     id: PropTypes.string
   };
 
   static defaultProps = {
     onSuggestionsUpdateRequested: noop,
-    shouldRenderSuggestions: value => value.trim().length > 0,
+    shouldRenderSuggestions: value => value && value.trim().length > 0,
```

Comment:
Ok, I removed this and the test added for https://github.com/moroshko/react-autosuggest/pull/165#discussion_r62436123 confirm that the `safeValue` change prevents this value from being null.